### PR TITLE
Fix "Duplicate contracts definition". fix #557

### LINKED ISSUE #557: Duplicate contracts definition
src/common/wallet/cointype.js
src/common/constants.js

### PR DESCRIPTION
**Pull request by imstar15**
pt的翻译文件有个冲突没注意到，被合并到develop里了，在这个PR里也修复一下。

### CODE REVIEW

File: src/common/translations/pt.json [hunk 1]
```diff
@@ -222,12 +222,8 @@
...
         "title": "Adicionar ativos personalizados",
         "address": "Endereço do contrato do ativo",
-<<<<<<< HEAD
-        "mainnet": "Rede principal"
-=======
-        "mainnet": "Mainnet",
-        "testnet": "Testnet"
->>>>>>> develop
+        "mainnet": "Rede principal",
+        "testnet": "Rede de teste"
```
Comment (bitcoinuser) [inline src/common/translations/pt.json:226]:
```suggestion
        "testnet": "Rede de testes"
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change Portuguese Brazilian

File: src/common/translations/pt.json (hunk 1 above)
Comment (imstar15) [reply src/common/translations/pt.json:226]: Portuguese Brazilian?
The file is pt.json.

**Review by chrisli30 — APPROVED** [review]
Looks good to me.

**Pull request merged**